[Concept Entry]: DDoS attacks (#2785)
Codecademy/docs: content/cybersecurity/concepts/cyber-attack/terms/ddos-attack/ddos-attack.md

---
Title: 'DDoS Attack'

> Description: 'A Distributed Denial of Service(DDoS) attack occurs when a person uses sources from multiple online locations to harm a business's online operations.'

Reviewer: ```suggestion
Description: 'A Distributed Denial of Service(DDoS) attack occurs when resources from multiple online locations are leveraged to harm the online operations of an organization.'
```

Subjects:
  - 'Cybersecurity'
  - 'Computer Science'
Tags:
  - 'Cybersecurity'
  - 'Computer Science'
CatalogContent:
  - 'introduction-to-cybersecurity'
  - 'paths/cyber-resilience-risk-management'
  - 'paths/fundamentals-of-cybersecurity'
---

> A **DDoS** is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) where a company or organization's websites and servers are unable to function effectively due to their network services are disrupted in an attempt to access their resources until they are no longer needed, ultimately leading to said websites and servers to stop working altogether.

Reviewer: ```suggestion
A **DDoS**, or Distributed Denial of Service, is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) where a company or organization's websites and servers are overwhelmed by traffic as a result of a coordinated effort by a hacker or group. The attack may use a variety of methods to make or amplify requests to a system that can cause disruptions of service and damages due to the inability of the system(s) to handle the flood of malicious or false requests.
```

## Types of DDOS Attacks

> DDoS attacks typically come in three categories:

Reviewer: ```suggestion
DDoS attacks can be separated into three categories:
```

> A **DDoS**, or Distributed Denial of Service, is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) where a company or organization's websites and servers are overwhelmed by traffic as a result of a coordinated effort by a hacker or group. These individuals often to use botnets, devices they hijack without the owners' knowledge, as a weapon of choice. However, hackers may use other methods, such as amplifying requests to a system that can cause disruptions of service and damages due to the inability of the system(s) to handle the flood of malicious or false requests.

Reviewer 2: ```suggestion
A **DDoS**, or Distributed Denial of Service, is a cyber attack where a company or organization's websites and servers are overwhelmed by traffic as a result of a coordinated effort by a hacker or group. These individuals often to use botnets, devices they hijack without the owners' knowledge, as a weapon of choice. However, hackers may use other methods, such as amplifying requests to a system that can cause disruptions of service and damages due to the inability of the system(s) to handle the flood of malicious or false requests.
```
Generally don't link to parent concept.

> 1. Volumetric(raw attack volume) attacks: Utilizes methods to generate volumes of online traffic to a company's website, causing its' bandwidth to be saturated and the flow of online viewership to be disjointed.

Reviewer: ```suggestion
1. Volumetric(raw attack volume) attacks: Utilizes methods to generate volumes of online traffic to a company's website, causing its' bandwidth to be saturated and destabilizing the flow of online accessibility.
```

> 2. Protocol attacks: Destroys the processing capacity of network infrastructure resources like servers, firewalls, and load balancers. This happens when malicious connection requests target by Layer 3 and 4 protocol connection requests.

Reviewer: ```suggestion
2. Protocol attacks: Destroys the processing capacity of network infrastructure resources like servers, firewalls, and load balancers. This happens when malicious connection requests are made through Layer 3 and 4 protocol connection requests.
```

> 1. Volumetric(raw attack volume) attacks: Utilizes methods to generate volumes of online traffic to a company's website, causing its' bandwidth to be saturated and destabilizing the flow of online accessibility.

Reviewer 2: ```suggestion
1. Volumetric(raw attack volume) attacks: Utilizes methods to generate volumes of online traffic to a company's website, causing its bandwidth to be saturated and destabilizing the flow of online accessibility.
```

> 3. Application Attacks: Opens connections and initiating process and transaction requests that consume finite resources like disk space and memory storage. This occurs when weaknesses in the application layer are manipulated

Reviewer: ```suggestion
3. Application Attacks: Opens connections and initiating process and transaction requests that consume finite resources like disk space and memory storage. This occurs when weaknesses in the application layer are manipulated.
```

> ## Signs of a DDOS-infected device

Reviewer: It might be helpful to have a sentence here, or in the first section, describing more explicitly what a botnet is and how they are used in DDOS attacks.

- Slow access to files.
- Unable to access a specific website.
- No internet access.

> - Continuous appearances of spam emails.

Reviewer: ```suggestion
- Spam emails to contacts.
```

## How to Respond to a DDOS Attack

- Provide training for staff members to recognize the signs of a DDoS attack.
- Use online outage mitigation and response strategies.
- Assess ongoing DDoS risks through tests that mimic these attacks.

> - Implement the technological tools that can handle the overflow of high Web traffics. Cloud-based Web servers like [AWS Shield](https://aws.amazon.com/shield/) are often recommended for this task.

Reviewer: ```suggestion
- Implement specialized tools that can handle the overflow of high Web traffic. Cloud-based Web servers like [AWS Shield](https://aws.amazon.com/shield/) are often recommended for this task.
```
